### ISSUE TITLE
Accessibility: MAS42A: Links are not visible as per the High contrast black/white themes.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#13]
#A11yMAS
#MAS42A
#Wipro
#vscode-website
#Firefox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Prerequisite:**
**_1. Change system setting to High contrast White_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast White" from the "Choose a theme" dropdown. Click on Apply.
**_2. Change system setting to High contrast Black_**
    Go to Settings-->Ease of Access-->High Contrast.
    Select "High Contrast Black" from the "Choose a theme" dropdown. Click on Apply.

**Repro Steps:**
**_Scenario-1:_**
1. Open [https://code.visualstudio.com/](https://code.visualstudio.com/) in Edge.
2. Navigate to the package download links below the "Get started now" above the footer of the page.
3. Observe the links in high contrast black/white theme.

**Note:** Same issue is observed for the below link
[http://code.visualstudio.com/Download](http://code.visualstudio.com/Download)

**_Scenario-2:_**
1. Open [http://code.visualstudio.com/insiders](http://code.visualstudio.com/insiders) in Edge.
2. Navigate to the "Download for Windows" link below the header of the page.
3. Observe the ".zip" link color in  high contrast black/white theme.

**Actual Result:**
**_Scenario-1:_**
The package download links are not visible as per the high contrast black/white theme.They are visible in white/black colors in HCB/HCW respectively.[refer screenshot]

**_Scenario-2:_**
The package download ".zip" link is not visible as per the high contrast black/white theme.They are visible in white/black colors in HCB/HCW respectively.[refer screenshot]

**Expected Result:**
**_Scenario-1:_**
The package download links should be visible in yellow/blue colors in HCB/HCW respectively.

**_Scenario-2:_**
The package download links should be visible in yellow/blue colors in HCB/HCW respectively.

**User Impact:**
The person who completely rely on high contrast theme, won't able to know that the elements are  "Links"

**Screenshots:**
[Scenario1_links are not visible as per HCBW theme.zip](https://github.com/wiproaccess/vscode-website/files/687054/Scenario1_links.are.not.visible.as.per.HCBW.theme.zip)
[Scenario2_Zip link color.zip](https://github.com/wiproaccess/vscode-website/files/687055/Scenario2_Zip.link.color.zip)

**Comment by waderyan** [comment]
![image](https://cloud.githubusercontent.com/assets/2449568/26735555/3d7e5c2c-4777-11e7-9192-3ed96cd1ef76.png)

![image](https://cloud.githubusercontent.com/assets/2449568/26735560/45428f82-4777-11e7-8676-ce4663633b1a.png)

![image](https://cloud.githubusercontent.com/assets/2449568/26735589/5fe297d8-4777-11e7-85cb-d621e14c6fca.png)

![image](https://cloud.githubusercontent.com/assets/2449568/26735591/65c82320-4777-11e7-8d31-2f3cf1368dfb.png)

**Comment by waderyan** [comment]
Added borders around the buttons. No need for the yellow text as they are now shown as buttons clearly. 

As with other areas of the site we need to use `href=javascript:void(0)` on the download links. I followed the same pattern here. 

Fixed with 548a8d7ef9fdb08caec131dbf35cadf4759377de

**Comment by wiproaccess** [comment]
The issue is verified hence adding the label 'closed' and closing the bug.
